.travis.yml

```diff
@@ -1,4 +1,16 @@
 language: python
 python: "2.7"
-install: "pip install -r requirements.txt"
-script: py.test
+
+addons:
+  apt:
+    packages:
+      - oracle-java8-installer
+      - oracle-java8-set-default
+
+install:
+  - pip install -r requirements.txt
+  - (cd Applications/SimpleUI; npm install)
```

Comment:
Haha, I didn't even think about that. I guess it's installed by default in Travis CI's Python container. The tricky part was getting the Java 8 VM set up so it would run htmllint.